challenges/objects-arrays.js

```diff
@@ -80,6 +111,9 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 
 // The zoos need a list of all their animal's names converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 let lowerCase = [];
+let lower = zooAnimals.map(animal =>{
```

Comment:
rather than returning here, you should be pushing the results of the looping into the lowerCase array, and then returning.